hdate/date.py

```diff
@@ -235,6 +235,23 @@ def omer_day(self):
             return 0
         return omer_day
 
+    @property
+    def daf_yomi(self):
+        """Return a tuple of mesechta and daf."""
+        days_since_start_cycle_11 = (self.gdate - htables.DAF_YOMI_CYCLE_11_START).days
+        daf_number = days_since_start_cycle_11 % (htables.DAF_YOMI_TOTAL_PAGES)
+        for mesechta in htables.DAF_YOMI_MESECHTOS:
+            if daf_number >= mesechta["pages"]:
+                daf_number -= mesechta["pages"]
+            else:
+                break
```

Comment:
Add +2 to daf number over here

Reply:
Please look at what I did in a6d7f7bce7455c7c608e35e2ebebf52238082df3 , I think it's clearer that way